### PR TITLE
feat: no blocked messages plugin

### PR DESCRIPTION
**Pull request by DiamondMiner88**
Hide all blocked messages from chat

### CODE REVIEW

File: src/plugins/noBlockedMessages.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    name: "NoBlockedMessages",
+    description: "Hides all blocked messages from chat completely.",
+    authors: [Devs.rushii],
+    patches: [
+        {
+            find: "safety_prompt:\"DMSpamExperiment\",response:\"show_redacted_messages\"",
+            replacement: [
+                {
+                    match: /collapsedReason;return \w{1,2}.createElement/,
+                    replace: "collapsedReason; return null; null.createElement"
```
Comment (Vendicated) [inline src/plugins/noBlockedMessages.ts:14]: null.createElement?

Comment (Vendicated) [reply src/plugins/noBlockedMessages.ts:14]: i mean it will work because you return before that but kinda weird xd

Comment (DiamondMiner88) [reply src/plugins/noBlockedMessages.ts:14]: You would still have to add some identifier so it parses correctly no?

File: src/plugins/noBlockedMessages.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export default definePlugin({
+    name: "NoBlockedMessages",
+    description: "Hides all blocked messages from chat completely.",
+    authors: [Devs.rushii],
+    patches: [
+        {
+            find: "safety_prompt:\"DMSpamExperiment\",response:\"show_redacted_messages\"",
+            replacement: [
+                {
+                    match: /collapsedReason;return \w{1,2}.createElement/,
```
Comment (Vendicated) [inline src/plugins/noBlockedMessages.ts:13]:
```suggestion
                    match: /collapsedReason;return (?=\w{1,2}.createElement)/,
```

File: src/plugins/noBlockedMessages.ts (hunk 1 above)
Comment (Vendicated) [inline src/plugins/noBlockedMessages.ts:14]:
```suggestion
                    replace: "collapsedReason; return null;"
```

**Review by Vendicated — APPROVED** [review]

**Pull request merged**